base/lnls-prune-artifacts.sh

```diff
@@ -77,7 +77,27 @@ get_used_epics_modules() {
     filter_out_paths "$all_modules" "$unused_modules"
 }
 
-remove_unused_epics_modules() {
+prune_module_dirs() {
+    module=$1
+
+    keep_paths="
+        $(find_shared_libs $module)
+        $(find $module -type f -regex ".*\.\(db\|template\|req\)" -printf "%h\n" | sort -u)
```

Comment:
Some IOCs use `substitutions` at runtime, I think they should also be kept...

Should ACFs be kept as well?